backend/src/services/mirroredModel/importers/image.ts

```diff
@@ -108,15 +131,7 @@ export class ImageImporter extends BaseImporter {
               },
               'Putting image blob.',
             )
-            await putImageBlob(
```

Comment:
Are we using this function anywhere else? Locally I can't see it being used other than here. If we no longer need it we need to consider removing it unless this is likely to be used in the future.